AlphaWallet/AppCoordinator.swift

```diff
@@ -268,50 +236,97 @@ class AppCoordinator: NSObject, Coordinator {
         WalletCoordinator(config: config, keystore: keystore, analyticsCoordinator: analyticsService).createInitialWalletIfMissing()
     }
 
+    private func showTransactionsIfNeeded(instantiateRequired: Bool = false) {
```

Comment:
What is `instantiateRequired` for? Seems not used.

Reply:
It doesn't needed, just experiments, will remove it later